native/audio-host/README.md

```
@@ -158,6 +163,19 @@ mac/build.sh
 `resources/bin/darwin-arm64` or `darwin-x64` depending on the build machine's architecture.
 As on Windows the copy is part of the build, not a step to remember.
 
+### Verify
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)